docs/development/new-kubernetes-version.md

```diff
@@ -0,0 +1,122 @@
+# Adding Support For A New Kubernetes Version
```

Comment:
Reference this in https://github.com/gardener/gardener/blob/master/docs/usage/supported_k8s_versions.md?